package.json

```diff
@@ -13,7 +13,8 @@
     "semantic-release": "semantic-release",
     "storybook": "start-storybook -p 6006",
     "storybook:build": "build-storybook",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "storybook:deploy": "storybook-to-ghpages -o .out",
+    "test": "npm run lint:js && npm run lint:styles && npm run prettier && npm run storybook:build"
```

Comment:
Why are we running `npm run storybook:build`?

Reply:
to verify that the tokens and everything will actually compile. I can't remember what the issue was, but I added this step when I realized there was broken build happening.